webapp/components/DonationInfo/DonationInfo.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="donation-info">
+    <progress-bar :title="title" :label="label" :goal="goal" :progress="progress" />
+    <a target="_blank" href="https://human-connection.org/spenden/">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-12T15:21:35Z" title="Tuesday, November 12th 2019, 4:21:35 pm +01:00">Nov 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="donation-info">
+    <progress-bar :title="title" :label="label" :goal="goal" :progress="progress" />
+    <a href="https://human-connection.org/spenden/">
```


```suggestion
    <a target="_blank" href="https://human-connection.org/spenden/">
```